commotionwireless.net/_includes/menu-navbar-main_fr.html

```diff
@@ -0,0 +1,79 @@
+                </ul>
+            </li>
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span class="glyphicon glyphicon-plus"></span> Télécharger</a>
+                <ul class="dropdown-menu">
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/download">Télécharger</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/download/routers">Routeurs</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/download/android">Téléphones</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/download/computers">Des ordinateurs</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/download/verify-signatures">Vérification des téléchargements</a></li>
+                </ul>
+            </li>
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span class="glyphicon glyphicon-plus"></span> Pour les organisateurs</a>
+                <ul class="dropdown-menu">
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/docs/cck">Introduction</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/docs/cck/planning">Planification</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/docs/cck/installing-configuring">Installation + Configuration</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/docs/cck/building-mounting">Bâtiment + montage</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/docs/cck/networking">Réseau sans fil + Réseau</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/docs/cck/local-applications">Applications locales</a></li>
+                </ul>
+            </li>
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span class="glyphicon glyphicon-plus"></span> Pour les développeurs</a>
+                <ul class="dropdown-menu">
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/developer/resources">Pour les développeurs</a></li>
+                    <li><a href="https://github.com/opentechinstitute">Code source</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/developer/api">Référence de l'API</a></li>
+                    <li><a href="https://wiki.commotionwireless.net">Wiki</a></li>
+                    <li><a href="https://github.com/issues?q=is:issue+user:opentechinstitute+is:open">Suivi du problème</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/developer/hig/introduction">Guide de l'interface humaine</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/jobs">Requests for Proposals</a></li>
```

Comment:
We had made this separate menu item 'requests for proposals' in the 'developer' section but it does link to the same place. I'm adding a translation for it now.